13.0.Dockerfile

```diff
@@ -210,19 +222,8 @@ ONBUILD ENV ADMIN_PASSWORD="$ADMIN_PASSWORD" \
             EMAIL_FROM="$EMAIL_FROM" \
             WITHOUT_DEMO="$WITHOUT_DEMO"
 ONBUILD ARG LOCAL_CUSTOM_DIR=./custom
-ONBUILD COPY $LOCAL_CUSTOM_DIR /opt/odoo/custom
-
-# Enable setting custom uids for odoo user during build of scaffolds
-ONBUILD ARG UID=1000
-ONBUILD ARG GID=1000
-
-# Enable Odoo user and filestore
-ONBUILD RUN groupadd -g $GID odoo -o \
-    && useradd -l -md /home/odoo -s /bin/false -u $UID -g $GID odoo \
-    && mkdir -p /var/lib/odoo \
-    && chown -R odoo:odoo /var/lib/odoo /qa/artifacts \
-    && chmod a=rwX /qa/artifacts \
-    && sync
+ONBUILD COPY --chown=root:odoo $LOCAL_CUSTOM_DIR /opt/odoo/custom
+ONBUILD RUN /opt/odoo/common/build.d/800-permissions
```

Comment:
If I'm not mistaken, you don't need this line, because it will get executed later when running `/opt/odoo/common/build`, right?

Also, I don't see it in other dockerfiles.

Reply:
yes, you are right, should be removed here as well.